cibuildwheel/windows.py

```diff
@@ -201,6 +202,13 @@ def pep_518_cp35_workaround(package_dir: Path, env: Dict[str, str]) -> None:
 
 
 def build(options: BuildOptions) -> None:
+    if options.architectures != [Architecture.AMD64, Architecture.x86]:
```

Comment:
Here's the source of the error I'm seeing when trying to split the windows build using `--archs`.
I know it could be done other ways but it might be good to be able to build individual architectures the same way on all OS. I know it was not the focus of the PR but would make configuration even easier (at least for my use case).

Reply:
What about `not set(option.architectures) <= {Architecture.AMD64, Architecture.x86}`?

Reply:
I will try to get this working in a branch. For the specific test, I guess you're right but I've not read the rest of the code which will probably require some adjustments.

Reply:
Meanwhile, the alternative would be to say "don't touch `--archs` on Windows, and just use `CIBW_BUILD`/`CIBW_SKIP`" (cfr. only one way to do things?)

Reply:
Using https://github.com/mayeut/cibuildwheel/commit/b5e620741f42d2ed3a62ae4c9d1a0fd301a15929 on top of this PR, I can achieve what I want: https://github.com/mayeut/pybase64/actions/runs/452918655

> Meanwhile, the alternative would be to say "don't touch --archs on Windows, and just use CIBW_BUILD/CIBW_SKIP" (cfr. only one way to do things?)

@YannickJadoul, "only one way to do things" makes sense however, although I understand why it would be specific for `qemu`, this PR now gives 2 ways of doing things on Linux (even if you're not using `qemu` specific arch) and if you're using `qemu`, then it easier to write your workflow configuration if this also works on Windows (at least on GHA). Not only it's easier, I find it conveys a very clear message on what is being achieved.

With `--archs` for all builds: 
```
    name: Build ${{ matrix.archs }} wheels on ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        include:
          - os: ubuntu-18.04
            archs: "x86_64,i686"
          - os: ubuntu-18.04
            archs: "aarch64"
          - os: ubuntu-18.04
            archs: "ppc64le"
          - os: ubuntu-18.04
            archs: "s390x"
          - os: windows-2019
            archs: "AMD64"
          - os: windows-2019
            archs: "x86"
          - os: macos-10.15
            archs: "x86_64"

    steps:
      ...
      - name: Build wheels
        env:
          CIBW_TEST_REQUIRES: "pytest"
          CIBW_TEST_COMMAND: "pytest --pyargs pybase64"
          CIBW_SKIP: "?p27* pp*"
        run: python -m cibuildwheel --archs ${{ matrix.archs }} --output-dir wheelhouse
```

vs
With `--archs` for all builds except Windows which uses `skip`:
```
name: Build ${{ matrix.archs }} wheels on ${{ matrix.os }} with skip ${{ matrix.skip }}
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        include:
          - os: ubuntu-18.04
            archs: "x86_64,i686"
            skip: ""
          - os: ubuntu-18.04
            archs: "aarch64"
            skip: ""
          - os: ubuntu-18.04
            archs: "ppc64le"
            skip: ""
          - os: ubuntu-18.04
            archs: "s390x"
            skip: ""
          - os: windows-2019
            archs: "auto"
            skip: "*win32*"
          - os: windows-2019
            archs: "auto"
            skip: "*win_amd64*"
          - os: macos-10.15
            archs: "x86_64"
            skip: ""

    steps:
      ...
      - name: Build wheels
        env:
          CIBW_TEST_REQUIRES: "pytest"
          CIBW_TEST_COMMAND: "pytest --pyargs pybase64"
          CIBW_SKIP: "?p27* pp* ${{ matrix.skip }}"
        run: python -m cibuildwheel --archs ${{ matrix.archs }} --output-dir wheelhouse
```